docs/guide/acquiring_data/jsoc.rst

```diff
@@ -0,0 +1,295 @@
+
+This returns an `~sunpy.net.fido_factory.UnifiedResponse` object containing
+information on the available online files which fit the criteria specified by
+the attrs objects in the above call. It does not download the files.
+
+To see a summary of results of our query, simple type the name of the
+variable set to the Fido search, in this case, res::
+
+    >>> res
+
+Now, let's break down the arguments of ``Fido.search`` to understand
+better what we've done.  The first argument:
+
+    ``a.jsoc.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00')``
+
+sets the start and end times for the query (any date/time
+format understood by SunPy's :ref:`parse_time function <parse-time>`
+can be used to specify dates and time). The Time attribute takes UTC time,
+as default. If you need to pass a Time in some other time scale, such as TAI,
+pass an Astropy Time object, like:
+
+	>>> from astropy.time import Time as T
+
+	``a.jsoc.Time(T('2014-01-01T00:00:00', scale='tai'), T('2014-01-01T01:00:00', scale='tai'))``
+
+The second argument:
+
+    ``a.jsoc.Series('hmi.v_45s')``
+
+sets the series we are looking for.
+
+So what is going on here?
+The notion is that a JSOC query has a set of attribute objects -
+described in ``a.jsoc`` - that are specified to construct the query.
+
+``a.jsoc.Series()`` is compulsory to be provided in each of the jsoc queries. Apart from this,
+atleast one prime-key must be passed (generally ``a.jsoc.Time()``).
+
+Querying with other PrimeKeys
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Other than Time, one other PrimeKey is supported with in-built attribute.
+In case of AIA series, ``a.jsoc.Wavelength()`` can be passed as a prime-key.
+
+    >>> import astropy.units as u	
+	>>> res = Fido.search(a.jsoc.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00'),
+						  a.jsoc.Series('aia.lev1_euv_12s'), a.jsoc.Wavelength(304*u.AA))
+
+Note that, only Time and Wavelength are in-built attributes here. If you need to pass any other primekey,
+it should be passed like this:
+
+	``a.jsoc.PrimeKey('HARPNUM', '4864')``
+
+	or, if 2 or more PrimeKeys need to be passed together:
+	``a.jsoc.PrimeKey('HARPNUM', '4864') & a.jsoc.PrimeKey('CAMERA', '2')``
+
+Also, note that the pre-defined primkeys, Time and Wavelength can also be passed as above, but you need to
+specify the exact keyword for it. For e.g. by :
+
+	``a.jsoc.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00'), a.jsoc.PrimeKey('WAVELNTH', '161')``
+
+If the correct keyword is not specified, or the passed PrimeKey is not supported by the given series, a
+meaningful error will be thrown, which will give you the primekeys supported by that series. Hence, by looking
+at the error, one can easily retry building the query with correct PrimeKeys.
+
+Other important thing to note is that, Wavelength when passed through in-built attribute, should be passed as a
+astropy quantity. Specifying spectral units in arguments is necessary or an error will be raised.
+To know more check `astropy.units`.
```

Comment:
I would re-word this to say something like 'for more information on units see...'